grant_access_to_anon.sql

```diff
@@ -0,0 +1,6 @@
+-- Grants access to the applicaton tables to the web_anon role for PostgREST usage
+GRANT SELECT ON application.tools TO web_anon;
+GRANT SELECT ON application.customers TO web_anon;
+GRANT SELECT ON application.rentals TO web_anon;
+
```

Comment:
Wouldn't it be better to grant access to all tables in both schemas for web_anon? (GRANT SELECT ON ALL TABLES IN SCHEMA application TO web_anon) If we add new tables and have to change the permissions every time it could be repetitive